README.md

```diff
@@ -426,6 +426,36 @@ To run test:
 mvn -Dtests.azure=true -Dtests.config=/path/to/config/file/elasticsearch.yml clean test
 ```
 
+Optimizing index storage on Azure File Service
+==============================================
+When using a shared file system based on the SMB protocol (like Azure File Service) to store indexes, the way Lucene open index segment files
+can slow down some functionalities like indexing and recovery. If you'd like to know more about this behavior and how Lucene and SMB interact,
```

Comment:
Wondering if we should say `like index and recovery` or `like indexing and recovering`